test/unit/dropin.js

```diff
@@ -1212,6 +1212,44 @@ describe('Dropin', () => {
       }
     );
 
+    test('calls verifyCardReady and setPaymentMethodRequestable when 3D secure is called', done => {
```

Comment:
```suggestion
    test('calls verifyCardReady and setPaymentMethodRequestable when 3D secure is complete', done => {
```

Reply:
let's add an expectation that `setPaymentRequestable` isn't being called before 3ds completes

Reply:
I'm thinking we may want tests for the following scenarios specific to 3DS if possible:

lookup and no step up challenge
lookup and subsequent step up challenge

Just to be doubly sure we're confident this event is being sent in the right order